src/odemis/acq/test/stream_test.py

```diff
@@ -4261,6 +4261,12 @@ def on_im(im):
     def test_arpolarimetry(self):
         """Test StaticARStream with ARPolarimetryProjection projection."""
 
+        # Check whether the arpolarimetry package is available
+        try:
+            import arpolarimetryDEBUG
```

Comment:
Why `DEBUG`? I only have `import arpolarimetry` in my code in Odemis. Did something change?

Reply:
Sorry about this. I had added it temporarily to test the behaviour works fine when the module is missing (bogus name).
I've just pushed a new version with the proper module name back.